apps/src/templates/studioHomepages/SetUpMessage.jsx

```diff
@@ -68,10 +67,13 @@ const SetUpMessage = ({
   isRtl,
   headingText,
   descriptionText,
+  className,
   buttonText,
   buttonUrl,
+  buttonClass,
+  onClick,
 }) => (
-  <div style={styles.section}>
+  <div style={styles.section} className={className}>
```

Comment:
Unfortunately without these properties, it's just `<div>`s all the way down... even the `Button` component, which we have to click on.